### ISSUE TITLE
Disappearing of the table of contents

### ISSUE THREAD

**Issue by djcaesar9114** [texstudio-org/texstudio#3091]
### Environment
- TeXstudio: 4.5.2 (git 0.0.0++)
- Qt:  Qt version 5.15.3, compilé avec Qt 5.15.3 R
- OS:Ubuntu 22.04
- TeX distribution: texlive

### Expected behavior

I want the table of contents to always be visible.

### Actual behavior

Sometimes, the table of contents disappear only to show my file "xxx.tex", without anything below. If I restart, everything comes back to normal.

### How to reproduce

That is the problem, I don't know what causes it. I'm working on my document and at some point I realise the table of contents has disappeared.

**Comment by djcaesar9114** [comment]
I'm sure. See your image? I only have the first line on this window (for me it's "essai.tex"), and nothing below. It's as if everything below has been deleted, although everything is fine in my document.

**Comment by idamir** [comment]
duplicate #3086

**Comment by djcaesar9114** [comment]
Actually not, I tried to do what he did, it doesn't work that way and the effects don't seem to be the same.

**Comment by idamir** [comment]
After you edit the string with \section{Lalala}, will you get the structure below "xxx.tex"?

**Comment by djcaesar9114** [comment]
> After you edit the string with \section{Lalala}, will you get the structure below "xxx.tex"?

Yes, the structure below "xxx.tex" updates when I change it, as it does for subsections. Actually, it seems pretty random. I'll try to notice a pattern and if I do, I'll post an update here.

**Comment by sunderme** [comment]
b4a01d1a5e04b6756828cb14ca30ebb97df48e10 may be a fix.
